bindings/src/main/java/io/pravega/storage/azure/AzureBlobClientImpl.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.storage.azure;
+
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.BlobServiceClientBuilder;
+import com.azure.storage.blob.models.BlobStorageException;
+import com.azure.storage.blob.models.BlobErrorCode;
+import com.azure.storage.blob.models.AppendBlobItem;
+import com.azure.storage.blob.models.BlobRequestConditions;
+import com.azure.storage.blob.models.BlobProperties;
+import com.azure.storage.blob.models.BlobRange;
+import com.azure.storage.blob.models.AppendBlobRequestConditions;
+import com.azure.storage.blob.specialized.AppendBlobClient;
+import com.azure.storage.blob.specialized.BlobClientBase;
+import com.azure.storage.common.StorageSharedKeyCredential;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+
+import java.io.InputStream;
+
+@Slf4j
+public class AzureBlobClientImpl implements AzureClient {
+    private final BlobContainerClient blobContainerClient;
+    private final AzureStorageConfig config;
+
+    public AzureBlobClientImpl(AzureStorageConfig config) {
+        this.config = config;
+        BlobServiceClient storageClient = new BlobServiceClientBuilder()
+                .endpoint(config.getEndpoint())
+                .credential(StorageSharedKeyCredential.fromConnectionString(config.getConnectionString()))
+                .buildClient();
+        this.blobContainerClient = storageClient.getBlobContainerClient(config.getContainerName());
+        log.debug("creating container {}.", config.getContainerName());
```

Comment:
Is this going to be called only once upon initialization? If so, it could be `info` level. Also, in general comments or log messages start by capitals.

Reply:
Yes, it would be called once for every run.